js/polyfills.js

```diff
@@ -0,0 +1,2 @@
+// Copy jquery sizes into source and require - so that it will 100% extend the same jquery instance instead of potentially a different one (due to different versions)
```

Comment:
I believe @JCCR addressed this in the Evident Point fork, right? (`package.json` uses `"jquery-sizes": "github:JCCR/jquery-sizes#master"`)

Reply:
its probably the fact that im using yarn and didn't know that yarn.lock was out of date (saw your other comment).

regardless, i feel what i've done is more reliable, the effective jquery dependency could change at any point and the runtime error may not be picked up before shipping...not sure how good the tests are.